Backpack/src/main/java/net/skyscanner/backpack/slider/BpkSlider.kt

```diff
@@ -0,0 +1,77 @@
+/**
+ * Backpack for Android - Skyscanner's Design System
+ *
+ * Copyright 2018-2021 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.skyscanner.backpack.slider
+
+import android.content.Context
+import android.content.res.ColorStateList
+import android.util.AttributeSet
+import android.view.ContextThemeWrapper
+import androidx.core.content.ContextCompat
+import com.google.android.material.slider.RangeSlider
+import net.skyscanner.backpack.R
+import net.skyscanner.backpack.util.BpkTheme
+import net.skyscanner.backpack.util.createContextThemeWrapper
+import net.skyscanner.backpack.util.use
+
+open class BpkSlider @JvmOverloads constructor(
+  context: Context,
+  attrs: AttributeSet? = null,
+  defStyleAttr: Int = 0
+) : RangeSlider(
```

Comment:
Using `RangeSlider` here, but I've added the value setter/getter/property which is the only difference between the range + default slider, so we can just have one component, rather than two